app/js/lib/view_handler/ppl_recurrent_warning.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var objectAssign = require( 'object-assign' ),
+
+	/**
+	 * View Handler for showing and hiding elements if the update value matches a regular expression
+	 * @class
+	 */
+	PPLRecurrentWarning = {
+		element: null,
+		isHidden: null,
+
+		update: function ( formContent ) {
+
+			if ( formContent.paymentType === 'PPL' && formContent.paymentIntervalInMonths > 0 ) {
+				if ( this.isHidden === false ) {
+					return;
+				}
```

Comment:
I copied that code from the `VisibilitySwitcher` class where it is needed to avoid unneccessary animations. I need to refactor this class to use the `Animator` interface.